tests/unit/lms/views/api/canvas/sync_test.py

```diff
@@ -15,6 +15,7 @@ def test_sync_when_the_user_is_a_learner(
         factories.HGroup(
             name="Section 2",
             authority_provided_id="d99674b9700f4a40a2b301d2949b61339c58236c",
+            type="section_group",
```

Comment:
There's no "type" field in the corresponding factory class so I'm guessing factoryboy just lets you add any extra params you send?

For compatibility with the underlying object, it might be nice if it had the same default of `course_group`.

Reply:
I actually hadn't considered the factory but yes it does seem to "just work" and the default from the model applies:

```python
>>> from tests import factories
>>> factories.HGroup()
HGroup(name='Test Group 0', authority_provided_id='166be8f2ded4070d27f993c52ad833ffe58d31b5', type='course_group')
>>> factories.HGroup(type="foo")
HGroup(name='Test Group 1', authority_provided_id='de2de2d96421ebde0da4a466c05045fff210f481', type='foo')
```